eng/pipelines/templates/jobs/ci.mgmt.yml

```diff
@@ -66,9 +63,6 @@ jobs:
       - template: /eng/pipelines/templates/steps/install-dotnet.yml
```

Comment:
Assuming you don't put the caching in the install-dotnet.yml file we should probably add the cache to the mgmt pipelines as well. Assuming @allenjzhang @m-nash @markcowl agree.